interface/billing/sl_eob_invoice.php

```diff
@@ -501,6 +501,12 @@ function updateFields(payField, adjField, balField, coPayField, isFirstProcCode)
                         <input type='text' name='form_stmt_count' id='form_stmt_count' class="form-control"
                                value='<?php echo attr((0 + $ferow['stmt_count'])); ?>'/>
                     </div>
+                    <div class="col-xs-2">
+                        <label class="control-label" for="form_last_bill"><?php echo xlt('Last Bill Date'); ?>
+                        :</label>
+                        <input type='text' name="form_last_bill" id='form_last_bill' class="form-control"
+                               value ='<?php echo attr($billdate); ?>' disabled/>
+                    </div>
```

Comment:
This is odd since is already in the main codebase.

Reply:
woops, i branched off of that last commit which obviously caused this

Reply:
branched off last commit which in prob not kosher